From: Luca Weiss <luca.weiss@fairphone.com>
Date: Fri, 26 May 2023 15:03:22 +0200
Subject: WIP? arm64: dts: qcom: sc7280: Fix irq types

Like on FP4

Use types that are hardcoded requested in the drivers there.

Avoid errors like
[    2.178003] irq: type mismatch, failed to map hwirq-14 for interrupt-controller@b220000!

See also https://lore.kernel.org/lkml/8735gi5g7s.wl-maz@kernel.org/

Signed-off-by: Luca Weiss <luca.weiss@fairphone.com>
---
 arch/arm64/boot/dts/qcom/sc7280.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/qcom/sc7280.dtsi b/arch/arm64/boot/dts/qcom/sc7280.dtsi
index a8cdc02..a197661 100644
--- a/arch/arm64/boot/dts/qcom/sc7280.dtsi
+++ b/arch/arm64/boot/dts/qcom/sc7280.dtsi
@@ -2920,7 +2920,7 @@ remoteproc_mpss: remoteproc@4080000 {
 			compatible = "qcom,sc7280-mpss-pas";
 			reg = <0 0x04080000 0 0x10000>;
 
-			interrupts-extended = <&intc GIC_SPI 264 IRQ_TYPE_EDGE_RISING>,
+			interrupts-extended = <&intc GIC_SPI 264 IRQ_TYPE_LEVEL_HIGH>,
 					      <&modem_smp2p_in 0 IRQ_TYPE_EDGE_RISING>,
 					      <&modem_smp2p_in 1 IRQ_TYPE_EDGE_RISING>,
 					      <&modem_smp2p_in 2 IRQ_TYPE_EDGE_RISING>,
